x/data/iri.go

```diff
@@ -120,24 +120,24 @@ func ParseIRI(iri string) (*ContentHash, error) {
 	const regenPrefix = "regen:"
 
 	if !strings.HasPrefix(iri, regenPrefix) {
-		return nil, ErrInvalidIRI.Wrapf("can't parse IRI %s without %s prefix", iri, regenPrefix)
+		return nil, ErrInvalidIRI.Wrapf("failed to parse IRI %s, %s prefix required", iri, regenPrefix)
```

Comment:
```suggestion
		return nil, ErrInvalidIRI.Wrapf("failed to parse IRI %s: %s prefix required", iri, regenPrefix)
```

nit: imo error messages look a bit better when the reason is trailed by a `:` rather than `,`

Reply:
I updated to use `:` except for when the error includes `, expected`, which I think a `,` still makes sense. https://github.com/regen-network/regen-ledger/pull/950/commits/a16c57abf82937abe9b194f73ce8632e508503a2